### ISSUE TITLE
Can add a share feature to the link that is created. 

### ISSUE THREAD

**Issue by aryan-139** [mfts/papermark#16]
I have been going through Papermark and right after creating the link. It might be good if we give options to share the link directly embedded as a Tweet. 

We can start with a Tweet first, then expand the sharing horizons as the need arises. 

I can work on this!

**Comment by mfts** [comment]
That is a great idea. A share interface would be really actionable. 

I'm pushing a PR today that enhances the upload/link creation and based on that it would be great to add a share function.

**Comment by 1nusah** [comment]
Hello, please confirm if this is done. I'd like to contribute

**Comment by mfts** [comment]
@1nusah this is not implemented yet. Feel free to contribute that feature :)

**Comment by surya-nand** [comment]
Hey Mark, I would like to contribute

**Comment by mfts** [comment]
@surya-nand Yes absolutely. I will release a roadmap this weekend and we get can started contributing together. If you already have something in mind let me know.

**Comment by surya-nand** [comment]
Along with an embedding link to share it as a tweet or text. I feel the link should look more credible, receivers should know what they are going to open.

•`https://papermark.io/view/${userName}/${documentName}${uniqueID}`( This will look more credible than below )
•https://papermark.io/view/clmbv79gp0002mj08o394lgcq

What do you think Mark?

**Comment by 1nusah** [comment]
This makes sense especially on the credibility bit

**Comment by mfts** [comment]
> https://papermark.io/view/${userName}/${documentName}${uniqueID}

This is a good suggestion. It would elongate the link a lot by a lot. Instead we should shorten it by introducing custom slugs like
- `https://papermark.io/view/${uniqueSlug}`
- `https://papermark.io/${uniqueSlug}`

So that we have either a random slug like "/cdhhdkeonq123jdk" or a personalized one like "/presentation"

This is already possible for custom domains. Shouldn't be too difficult to port that over.

**Comment by surya-nand** [comment]
As you said earlier, drop your roadmap this week 
We can contribute together.

**Comment by Kurama-nine** [comment]
Is it available as yet?
